tradetrust/document-store-worker/src/worker/__init__.py

```diff
@@ -33,7 +33,8 @@ def __init__(self, config=None):
 
     def eth_connect(self):
         logger.debug('eth_connect')
-        self.web3 = Web3(Web3.HTTPProvider(self.config['Blockchain']['Endpoint']))
+        self.web3 = Web3(Web3.HTTPProvider(config['Endpoint']))
```

Comment:
the config changed (no self and lo blockchain anymore) and I'm not sure if it was deliberate change

Reply:
Thanks fixed it up, I had made a local config var for the Blockchain side but ended up removing it